client/entrypoints/information-overlay/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import InformationOverlay from './information-overlay';
+
+const stripeRowTop = jQuery( 'tr[data-gateway_id="stripe"]' ).offset().top;
+const windowHeight = jQuery( window ).height();
+
+// waiting for the dom to be fully loaded as the section below the table takes time to load
+jQuery( () => {
+	const scrollTop =
+		stripeRowTop > windowHeight / 2
+			? stripeRowTop - windowHeight / 2
+			: stripeRowTop;
+	// scrolling so that the Stripe row is always within view
+	jQuery( 'body,html' ).animate( { scrollTop }, 800, () => {
+		const informationOverlayContainer = document.getElementById(
+			'wc-stripe-information-overlay-container'
+		);
+
+		if ( informationOverlayContainer ) {
```

Comment:
If the container is not present in the DOM, the Stripe row would be highlighted and the window would be scrolled.

What do you think about putting this check at the top?

(very minor) We could get rid of all this jQuery code and just use vanilla JS to assign the styles, in a `useEffect` on mount of the `InformationOverlay` component.